resultsdb/messaging.py

```diff
@@ -152,6 +154,9 @@ class DummyPlugin(MessagingPlugin):
     history = []
 
     def publish(self, message):
+        # Add telemetry information. This includes an extra key
+        # traceparent.
+        TraceContextTextMapPropagator().inject(message)
```

Comment:
This won't be triggered in production. We use the `StompPlugin` in prod.

Reply:
Added to Stomp